### DISCUSSION TITLE
Vim repeating "d" button

### DISCUSSION THREAD

**Discussion by mtfu** [manna-harbour/miryoku#109] [Q&A] [answered]
Hi thanks for sharing and creating this lovely layout.
Such a joyful typing experience using this layout!

I am currently used the following layout [Miryoku Oryx (Moonlander) changed to use qwerty](https://configure.zsa.io/moonlander/layouts/bJ5n5/G7Drg/0)

I am having trouble with repeating the "d" button which is use quite a lot with in vim with ctrl + d(move forward 1/2 a screen), for navigation.
"d" is also used in Vimium chrome extension, for scrolling down.

Does anyone have a helpful tip or trick, for repeating "d" without triggering ctrl mode.

**Comment by manna-harbour** [comment, marked as answer]
Use the cursor keys and page up / down on Nav instead, as they auto repeat. See https://github.com/manna-harbour/miryoku/discussions/105#discussioncomment-3105056. Also, they won't change position if you change to a different alpha arrangement.

Alternatively, enable tap-hold auto repeat. See https://github.com/manna-harbour/miryoku/discussions/73.

Or you could add `d` to an unused position. That's easy in Miryoku Oryx, but for other implementations see https://github.com/manna-harbour/miryoku/discussions/85.

**Reply by mtfu** [reply]
Thank you for your response, I use the Nav menu for navigation with hjkl in Vim and it works perfectly.
I tried to use d on a unused position, but did not do the trick for me.

For now I will just repeat d, or maybe map the key to something different in vim :)
